src/theme/theme.js

```diff
@@ -14,6 +14,8 @@ export const baseTheme = {
     h3: "1.3rem",
     defaultLarger: "1.5rem",
     large: "2.5rem",
+    xlarge: "3em",
```

Comment:
Typically we don't need new sizes you can normally add it in the styled section so
```
const NameOfComponent = styled(Text)`
font-size: 3em;
`;
```

We only add sizes to theme if we feel like we're going to be reusing it a lot and I'm not sure if we will since it is in em and not rem. Just a thought